aws/eip.go

```diff
@@ -0,0 +1,94 @@
+package aws
+
+import (
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func getOrSetFirstSeenTag(svc *ec2.EC2, address ec2.Address) (time.Time, error) {
```

Comment:
I'm not a fan of a method that does two things. It's harder to reason about it and harder to test. If you break it up into two methods—`getFirstSeenTag` and `setFirstSeenTag`—it's clearer what each does and you can have unit tests for each.

Reply:
Also, please add a comment to this method explaining how and why we tag EIPs